NodeTypes/Collection/Content/Main.yaml

```diff
@@ -10,3 +10,4 @@
     nodeTypes:
       '*': false
       'Neos.Demo:Constraint.Content.Main': true
+      'Neos.Demo:Content.CarouselYouTube': false
```

Comment:
hmm we are using this pattern here to have the `Neos.Demo:Constraint.Content.Main` be a mixin - so as long as the `Neos.Demo:Content.CarouselYouTube` doesnt extend this mixin - it should be fine

see https://docs.neos.io/tutorials/better-constraints